core-plugins/src/e2e-test/features/fileplugin/source/FileToFile.feature

```diff
@@ -0,0 +1,405 @@
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "File" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "textFile"
+    Then Select dropdown plugin property: "format" with option value: "text"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "textFileSchema"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "File2"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "filePluginOutputFolder"
+    Then Replace input plugin property: "pathSuffix" with value: "yyyy-MM-dd-HH-mm"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Click on the Preview Data link on the Sink plugin node: "File"
+    Then Verify sink plugin's Preview Data for Input Records table and the Input Schema matches the Output Schema of Source plugin
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate output file generated by file sink plugin is equal to expected output file "outputForTextInputTest"
+
+  Scenario: To verify data is getting transferred from File source to File sink with Path Field
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "File" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "File" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "outputFieldTestFile"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Enter input plugin property: "pathField" with value: "outputPathField"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "outputFieldTestFileSchema"
```

Comment:
nit : Rename `outputField` to `pathField` everywhere.

Reply:
Renamed.